README.md

```diff
@@ -1 +1,97 @@
-# cron-command
+# EasyEngine/cron-command
+
+Manages cron jobs in EasyEngine
+
+`cron` command contains following subcommand
+ * [add](#add)
```

Comment:
Mention the full command like https://github.com/EasyEngine/site-command/blob/develop/README.md to maintain similarity

Reply:
@mbtamuli It was getting redundant. So I thought it's best to keep it short like this and update it like this in site-command too

Reply:
> update it like this in site-command too
Okay